### PR TITLE
Reformat typescript using prettier

### PR DESCRIPTION
**Pull request by ssbarnea**
That enables typescript on pre-commit prettier run as currently almost
any attempt to edit .ts files using vscode would introduce undesired
reformatting. That is because we recommend prettier extension which
auto-reformats.

The only manually made change is the one line in pre-commit config,
the rest is just pre-commit itself.

Sister change: https://github.com/ansible/ansible-language-server/pull/196
Related: https://github.com/ansible-community/devtools/issues/28

### CODE REVIEW

File: src/features/runner.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /* "stdlib" */
 import * as vscode from 'vscode';
 
-
```
Comment (webknjaz) [inline src/features/runner.ts:4]: Keep a two-line separator after the imports section.

File: src/features/runner.ts [hunk 2]
```diff
@@ -14,33 +13,33 @@ export class AnsiblePlaybookRunProvider {
...
+   * Register a set of command callbacks for executing Ansible playbooks
+   * within VS Code.
+   */
   private configureCommands() {
     this.vsCodeExtCtx.subscriptions.push(
       vscode.commands.registerCommand(
         'extension.ansible-playbook.run',
-        fileObj => this.makeCmdRunner()(fileObj),
-      ),
+        (fileObj) => this.makeCmdRunner()(fileObj)
```
Comment (webknjaz) [inline src/features/runner.ts:23]: The braces only add clutter, don't add them unnecessarily.

File: src/features/runner.ts [hunk 3]
```diff
@@ -51,106 +50,101 @@ export class AnsiblePlaybookRunProvider {
...
       }
       console.debug(
         `Got a request to run ansible-${
-          useNavigator
-            ? 'navigator'
-            : 'playbook'
+          useNavigator ? 'navigator' : 'playbook'
         } against`,
-        playbookFsPath,
+        playbookFsPath
```
Comment (webknjaz) [inline src/features/runner.ts:62]: Always keep trailing commas.

**Review by webknjaz — COMMENTED** [review]
I've identified a number of highly undesirable changes below. The config probably needs fixing before it's acceptable.

**Review by webknjaz — CHANGES_REQUESTED** [review]
(missed a button in the previous review)

File: src/features/runner.ts (hunk 3 above)
Comment (ssbarnea) [reply src/features/runner.ts:63]: Irrelevant because all changes are made by prettier, on save. There is nothing configurable about it.

**Review by priyamsahoo — APPROVED** [review]
LGTM

File: src/features/runner.ts (hunk 2 above)
Comment (ssbarnea) [reply src/features/runner.ts:23]: Watch https://github.com/ansible/vscode-ansible/pull/407 for an answer. Outside our control.

**Comment by ssbarnea** [comment]
There is on known bug https://github.com/prettier/prettier/issues/7976 that adds some extra parentheses but we can live with it, like we already did so far. Thus is why it was not fixes for a long time.

**Pull request merged**